statsmodels/survey/examples/survey_df.csv

```diff
@@ -0,0 +1,184 @@
+"dnum","pw","api00","api99"
```

Comment:
I'm not sure how Josef feels about this, but I would avoid committing examples at this point with large data sets.  Eventually we will want some examples with real survey datasets (e.g. MEPS, NHANES).  But we wouldn't want to bring that data into the statsmodels repo I think.  A large simulated dataset isn't that helpful as a demo in most cases.  Tests can usually get by with much smaller datasets that (ideally) don't need to be read from disk.  Right now, tests are more important than examples.